cves/CVE-2017-9793.yml

```diff
@@ -162,11 +162,14 @@ interesting_commits:
     If there are no interesting commits, demonstrate that you completed this section
     by explaining what happened between the VCCs and the fix.
   answer:
+    The commit I highlighted was very important for the fix for this vulnerability as
```

Comment:
I might split this up into two sentences. The sentence felt like it ran on for quite a while and was hard to follow.

Reply:
Thanks, will edit.